tests/mli/channel.py

```diff
@@ -54,6 +54,16 @@ def __init__(self, key: pathlib.Path) -> None:
 
         self._file_path.touch()
 
+    def __getstate__(self):
```

Comment:
ditto on dangerousness of removing the lock. Let's fix the channels instead

Reply:
removed!